test/bravery-components/braveryPanelTest.js

```diff
@@ -882,29 +882,6 @@ describe('Bravery Panel', function () {
               return body === ''
             })
         })
-
-        .keys(Brave.keys.ESCAPE)
```

Comment:
introduced here: https://github.com/brave/browser-laptop/pull/8990/files#diff-f94f0ea4252431046502bbe75728ba1fR836. I don't think it is needed for this test specifically